Gemfile

```diff
@@ -60,10 +58,10 @@ group :test, :development do
 end
 
 gem "acts_as_list"
-gem 'aws-sdk', '= 2.0.22'
+gem 'aws-sdk', '~> 3'
```

Comment:
I'm wondering if the major version release is going to break any editor code, like the rubric saving.  We'll see on staging.

Reply:
No problems saving rubrics on staging!  Nice job.